<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>searchPanel</class>
 <widget class="QFrame" name="searchPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>33</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QComboBox" name="columnBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>171</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="conditionBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>61</width>
     <height>29</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>LIKE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>=</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="patternEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>searchPanel</receiver>
   <slot>deletePanel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>448</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>deletePanel()</slot>
 </slots>
</ui>
